### DISCUSSION TITLE
MVT High?

### DISCUSSION THREAD

**Discussion by ghost** [hooke007/mpv_PlayKit#441] [z_ENG_discuss]
Hello, I was reading this [gist](https://gist.github.com/phiresky/4bfcfbbd05b3c2ed8645) and this block seemed interesting.
```
    if mpix > 1.5:
        # can't handle these on Full HD with Intel i5-2500k
        # see the description of these parameters in http://avisynth.org.ru/mvtools/mvtools2.html#functions
        analParams['search'] = 0
        blockParams['mode'] = 0
        quality = 'low'
    else:
        quality = 'high'
```
Does your MVT STD use this quality variable? If no, how can we use the quality=high.
Sincerely.

**Comment by hooke007** [comment]
>Does your MVT STD use this quality variable?

Isn't it just a printed text?
```
print("Reflowing from {} fps to {} fps (quality={})".format(clip.fps_num/clip.fps_den,dst_fps_num/dst_fps_den,quality))
``

**Comment by ghost** [comment]
lol, yeah mb

**Comment by hooke007** [comment]
BTW, the LQ/STD may not be the accurate description. In some cases LQ should be better.

**Reply by ghost** [reply]
Thank you, I decided to use Rife 4.15 lite, it felt its better.
